packages/nuxt3-module/plugin.ts

```diff
@@ -11,19 +11,21 @@ import { createAPIClient } from "@shopware/api-client";
 
 export default defineNuxtPlugin((NuxtApp) => {
   const runtimeConfig = useRuntimeConfig();
-
-  if (
-    !runtimeConfig.public?.shopware?.shopwareEndpoint ||
-    !runtimeConfig.public?.shopware?.shopwareAccessToken
-  ) {
+  const shopwareEndpoint =
+    runtimeConfig.public?.shopware?.endpoint ??
+    runtimeConfig.public?.shopware?.shopwareEndpoint;
+  const shopwareAccessToken =
+    runtimeConfig.public?.shopware?.accessToken ??
+    runtimeConfig.public?.shopware?.shopwareAccessToken;
+  if (!shopwareEndpoint || !shopwareAccessToken) {
     throw new Error(
       "Make sure that shopwareEndpoint and shopwareAccessToken are settled in the configuration",
```

Comment:
great you've left the backward compatibility 💪🏼 

the error message should also be adjusted in this case, WDYT?